src/Microsoft.SqlTools.ManagedBatchParser/BatchParser/Parser.cs

```diff
@@ -541,7 +540,7 @@ private void ParseSetvar(Token setvarToken)
         }
 
 
-        private void ParseConnect(Token connectToken)
```

Comment:
In the future I would suggest suppressing new issues as long as they aren't causing actual compilation failures or something, and then doing follow up PRs to unsuppress and fix them. Makes it much easier to review and prevents issues from sneaking in.